pkg/common/schema/log.go

```diff
@@ -28,8 +28,11 @@ type JobLogInfo struct {
 }
 
 type TaskLogInfo struct {
-	TaskID string  `json:"taskID"`
-	Info   LogInfo `json:"logInfo"`
+	// container name
+	TaskID  string  `json:"taskID"`
+	PodName string  `json:"podName"`
```

Comment:
`TaskID` is equal to `PodUID` on kubernetes cluster.

Reply:
done